manifest.php

```diff
@@ -27,7 +27,7 @@
     'label' => 'Tao Offline',
     'description' => 'An extension to assist the setup of an offline context. Setup synchronisation and encryption by test center identifier',
     'license' => 'GPL-2.0',
-    'version' => '2.2.0',
+    'version' => '2.2.1',
```

Comment:
It sounds like more a minor change than a patch. What do you think?

Reply:
I think patch should be fine, it looks like a bug in current behavior